### PR TITLE
ReadMe Updates

### PR DESCRIPTION
**Pull request by EricVoll**

### CODE REVIEW

**Comment by EricVoll** [comment]
@hanlonm I read through the instructions again and noticed a few small details. Most notably is the info about the gazebo simulation (or that it's possible to run the app with a simulated robot), which is important since many people might not have a kinova arm standing around :) 

Please feel free to correct any of my changes.

**Comment by hanlonm** [comment]
@EricVoll Thanks for the feedback! All of the minor ReadMe updates look good to me. However, I noticed that the gazebo simulation does not work in the current state due to the changes I made to get pose correction working. For now I think it is best not to reference simulation directly in the ReadMe.

I will create a separate PR to add back the simulation functionality. It should only be some minor changes in the ROS code that I can test locally.

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ cd ~/catkin_ws
 rosdep install --from-paths src --ignore-src -r -y
 ```
 5. Build and source the workspace with `catkin build && source devel/setup.bash`
-6. Test the setup by running `roslaunch mrirac kinova_real.launch`
+6. Test the setup by running `roslaunch mrirac kinova_real.launch` or alternatively (if you don't have a Kinova arm available or setup) you can launch the Gazebo simulation using `roslaunch mrirac kinova_sim.launch`.
```
Comment (hanlonm) [inline README.md]: `roslaunch mrirac kinova_sim.launch` needs to be replaced with `bash MRIRAC_ROS/mrirac/launch_sim.bash`

**Review by hanlonm — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (EricVoll) [reply README.md]: Thanks!

**Review by hanlonm — APPROVED** [review]

**Pull request merged**